examples/webhook-authz/README.md

```diff
@@ -100,7 +103,7 @@ or create/update them as `testuser`:
 
 ```
 sh
-$ kubectl apply -f cluster.yaml --context <your_cluster_registry_context> --as testuser
+$ kubectl apply -f ../samplecontainer/cluster.yaml --context <your_cluster_registry_context> --as testuser
 Error from server (Forbidden): error when applying patch:
```

Comment:
The error I received was a `create` error and not a `patch` error since it was the first time created:
```
Error from server (Forbidden): error when creating "../samplecontainer/cluster.yaml": clusters.clusterregistry.k8s.io is forbidden: User "testuser" cannot create clusters.clusterregistry.k8s.io at the cluster scope: Not allowed
```

Reply:
Good point. Updated.